packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx

```diff
@@ -134,13 +134,19 @@ export function createStorageBrowser<
   const StorageBrowser: StorageBrowserType<
     DerivedActionViewType<RInput>,
     DerivedActionViews<RInput>
-  > = ({ views, displayText }) => (
-    <ErrorBoundary>
-      <Provider displayText={displayText} views={views}>
-        <StorageBrowserDefault />
-      </Provider>
-    </ErrorBoundary>
-  );
+  > = ({ views, displayText }) => {
+    const OptionalErrorBoundary =
+      input.ErrorBoundary === null
+        ? React.Fragment
+        : input.ErrorBoundary ?? DefaultErrorBoundary;
```

Comment:
Prefer to resolve `ErrorBoundary` outside the scope of the `StorageBrowser` function declaration